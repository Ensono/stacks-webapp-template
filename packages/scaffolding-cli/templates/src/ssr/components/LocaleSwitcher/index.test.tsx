import {render, act} from "@testing-library/react"
import React from "react"
import {LocaleSwitcher} from "."

test("renders LocaleSwitcher snapshot", async () => {
    let renderedComp
    await act(async () => {
        renderedComp = render(<LocaleSwitcher />)
    })
    expect(renderedComp.asFragment()).toMatchSnapshot()
})
