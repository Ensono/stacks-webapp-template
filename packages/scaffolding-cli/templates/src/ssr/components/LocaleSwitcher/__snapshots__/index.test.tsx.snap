// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ApiPane snapshot 1`] = `<DocumentFragment />`;
